### PR TITLE
Focus is put on error message X and then on update button after that

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js [hunk 1]
```diff
@@ -180,3 +181,37 @@ describe('Modals on the personal information and content page after editing', ()
     cy.findByLabelText(/email address/i);
   });
 });
+
+describe('Modals on the personal information and content page when they error', () => {
+  it('should allow the ability to reopen the edit modal when the transaction completes', () => {
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:186]: this `it` description doesn't seem right

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js [hunk 2]
```diff
@@ -180,3 +181,37 @@ describe('Modals on the personal information and content page after editing', ()
...
+    });
+
+    // expect an error to be shown
+    cy.findByTestId('edit-error-alert').should('exist');
+
+    // check for error modal and check that the close is focused then click it
+    cy.findByRole('button', { name: /close notification/i }).should(
+      'be.focused',
+    );
+    cy.findByRole('button', { name: /close notification/i }).click();
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:212]: can you just chain this `click()` to the end of the previous line, right after the `should()`?

File: src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx [hunk 3]
```diff
@@ -80,6 +80,29 @@ export class ContactInformationEditView extends Component {
     }
   }
 
+  focusOnUpdateButton() {
+    const focusableElement = this.editForm?.querySelector(
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx:84]: I think all this code could be replaced with `focusElement('[data-testid="save-edit-button"]')` after you `import { focusElement } from 'platform/utilities/ui';`

File: src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx [hunk 4]
```diff
@@ -80,6 +80,29 @@ export class ContactInformationEditView extends Component {
...
+      '[data-testid="save-edit-button"]',
+    );
+    if (focusableElement) {
+      focusableElement.focus();
+    }
+  }
+
+  focusOnError() {
+    const focusableElement = this.errorAlert?.querySelector(
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx:93]: same here. I think `focusElement('button[aria-label="Close notification"]')` should work

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js (hunk 1 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:186]: wow sorry about that. I should have noticed that.

File: src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx (hunk 3 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx:84]: I tried this I thought. I will try it out again

File: src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx [hunk 5]
```diff
@@ -260,11 +272,14 @@ export class ContactInformationEditView extends Component {
             role="alert"
             className="vads-u-margin-bottom--2"
             data-testid="edit-error-alert"
+            ref={el => {
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx:275]: is this still needed?

**Review by erikphansen — APPROVED** [review]
not sure if that ref is needed, but looks good!

**Review by erikphansen — APPROVED** [review]

**Pull request merged**